frontends/aiger/aigerparse.cc

```diff
@@ -722,7 +722,8 @@ void AigerReader::parse_aiger_binary()
 		module->connect(wire, createWireIfNotExists(module, l1));
 		outputs.push_back(wire);
 	}
-	std::getline(f, line); // Ignore up to start of next line
+	if (O > 0)
+		std::getline(f, line); // Ignore up to start of next line
```

Comment:
Thanks! Please move up the getline below `f >> l1` so we consume the full line per every output.

Reply:
Oh sure I can do that, will do the same for the following `if (B > 0)` and the ascii parser while im at it